zulipterminal/ui_tools/boxes.py

```diff
@@ -801,6 +807,8 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                     )
         elif is_command_key("GO_BACK", key):
             self.send_stop_typing_status()
+            if len(self.msg_write_box.edit_text) > 10:
```

Comment:
Generally these literal values (ie. like 10) are good to extract as named constants. That makes it clearer what the value is at the point where it is used, and also means that one doesn't necessarily need to search through the code to determine what a value/threshold is.